sys-power/powertop/powertop-9999.ebuild

```diff
@@ -96,6 +96,8 @@ pkg_setup() {
 
 src_prepare() {
 	if [[ ${PV} == "9999" ]] ; then
+		chmod +x scripts/version || die "Failed to make 'scripts/version' executable"
+		scripts/version || die "Failed to extract version information"
 		eautoreconf
```

Comment:
are you sure `src_prepare()` will exit cleanly? where do you call `eapply_user` when `${PV} == "9999"`?